gemfiles/README.md

```diff
@@ -0,0 +1,17 @@
+## Update Gemfiles
+The files in this directory are being used by [GitHub Actions rspec workflow](/.github/workflows/rspec.yml) to run the 
+test suite with different ActiveRecord and Ruby versions.
+
+### Update (with rbenv)
+1. Make sure you have the ruby versions listed in the [rspec workflow](/.github/workflows/rspec.yml) installed with rbenv.
+2. To update the .lock files, run:
+```shell
+RBENV_VERSION=3.0.5 BUNDLE_GEMFILE=ar_6_0.gemfile bundle install
```

Comment:
PS. This was added by an open pr in the normal README as well. https://github.com/zilverline/sequent/commit/f6b2467eb0facf2b3ee34d5362f1b11895fde6b2

Reply:
Ah I missed that. Should I remove the `README.md` in the `gemfiles` directory? 
I see now that my commands are a bit redundant as some overwrite the already generated .lock files.

Reply:
Yeah you can remove as it is duplicate now.